src/controller/task.spec.ts

```diff
@@ -0,0 +1,215 @@
+import { expect } from 'chai'
+import { after, before, describe } from 'mocha'
+import TaskModel from '../model/task'
+import axios from 'axios'
+// eslint-disable-next-line @typescript-eslint/no-unused-vars
+import app from '../app'
+
+const _server = app
+axios.defaults.baseURL = 'http://localhost:3000'
+
+const Tasks = {
+  _id: '6148502fe5e1d14502694f42',
+  description: 'test description',
+  title: 'node js learning',
+  importance: 'LOW',
+  status: 'NOT_STARTED',
+  completedAt: null,
+  updatedAt: '2021-09-28T14:37:13.659Z',
+  createdAt: '2021-09-28T14:37:13.659Z'
+}
+
+const validId = Tasks._id
+const invalidId = 777
+const nonExisted = '6148502fe5e1d14502694f56'
+
+describe('Tasks', () => {
+  describe('POST ', () => {
+    after('Delete test data', async () => {
+      await TaskModel.deleteMany()
+    })
+    it('should create a task', async () => {
+      const createTask = {
+        title: 'title TEST AXIOUS',
+        description: 'desc TEST AXIOUS'
+      }
+      const response = await axios.post('/tasks', createTask)
```

Comment:
Is better to have `const { status, data }` destructuring instead of chaining `response.data.title`